vegafusion-sql/tests/test_aggregate.rs

```diff
@@ -138,57 +143,59 @@ mod test_variance_aggs {
-                        args: vec![col("a")],
-                        distinct: false,
-                        filter: None,
-                    })
-                    .mul(lit(100)),
-                )
-                .div(lit(100))
-                .alias("var_a"),
-                round(
-                    Expr::AggregateFunction(expr::AggregateFunction {
-                        fun: AggregateFunction::VariancePop,
-                        args: vec![col("a")],
-                        distinct: false,
-                        filter: None,
-                    })
-                    .mul(lit(100)),
-                )
-                .div(lit(100))
-                .alias("var_pop_a"),
-            ],
-        );
-        let df_result = df_result.and_then(|df| {
+        let df = SqlDataFrame::from_values(&table, conn, Default::default()).unwrap();
+        let df_result = df
+            .aggregate(
+                vec![col("b")],
+                vec![
+                    round(
+                        Expr::AggregateFunction(expr::AggregateFunction {
+                            fun: AggregateFunction::Stddev,
+                            args: vec![col("a")],
+                            distinct: false,
+                            filter: None,
+                        })
+                        .mul(lit(100)),
+                    )
+                    .div(lit(100))
+                    .alias("stddev_a"),
+                    round(
+                        Expr::AggregateFunction(expr::AggregateFunction {
+                            fun: AggregateFunction::StddevPop,
+                            args: vec![col("a")],
+                            distinct: false,
+                            filter: None,
+                        })
+                        .mul(lit(100)),
+                    )
+                    .div(lit(100))
+                    .alias("stddev_pop_a"),
+                    round(
+                        Expr::AggregateFunction(expr::AggregateFunction {
+                            fun: AggregateFunction::Variance,
+                            args: vec![col("a")],
+                            distinct: false,
+                            filter: None,
+                        })
+                        .mul(lit(100)),
+                    )
+                    .div(lit(100))
+                    .alias("var_a"),
+                    round(
+                        Expr::AggregateFunction(expr::AggregateFunction {
+                            fun: AggregateFunction::VariancePop,
+                            args: vec![col("a")],
+                            distinct: false,
+                            filter: None,
+                        })
+                        .mul(lit(100)),
+                    )
+                    .div(lit(100))
+                    .alias("var_pop_a"),
+                ],
+            )
+            .await;
+        let df_result = if let Ok(df) = df_result {
```

Comment:
The `and_then` pattern we used before doesn't play well with async query methods, so restructure to use an `if let Ok` pattern.